mobify/progressive-web-scaffold
# Astro: HYB-984 (Setting up carts)

Add Cart View page to Merlin's demo app

 **JIRA**: [**HYB-984**](https://mobify.atlassian.net/browse/HYB-984)

## TODO
- [x] correct cart url
- [x] badge counter
- [x] setup with native headerbar

## Changes
- Added cart Icon and close Icon
- Added cartConfig that contains cart flow related data
- Added CartHeaderController : headerbar for checkout flow
- Added CartModalController : modal view with navigation for check flow
- Edited tabController : clicking the cartIcon opens up the modal view for checkout flow

## How to test-drive this PR
- Run the app (note that no web header or footer is displayed on any page)
- Click the cart button, it should open a modal webview with close button that leads to a random page (in the future should lead to cart page)
